### PR TITLE
Fix/design review

### PR DESCRIPTION
**Pull request by meganrm**
Problem
=======
#16 

Solution
========
Updates to style

preview: https://deploy-preview-19--cell-catalog.netlify.app/disease-catalog/

## Type of change
Please delete options that are not relevant.

* Bug fix (non-breaking change which fixes an issue)

Screenshots (optional):
-----------------------
Screenshot 2024-03-11 at 11 30 47 AM

### CODE REVIEW

File: src/components/Layout.tsx [hunk 1]
```diff
@@ -7,11 +7,11 @@ const { Content } = AntLayout;
 import Navbar from "./Navbar";
 import "../style/index.sass";
 import useSiteMetadata from "./SiteMetadata";
-import { container } from "../style/layout.module.css";
+const { container } = require("../style/layout.module.css");
 
 const OFF_WHITE = "#F2F2F2";
 const DARK_BLUE = "#00215F";
-const WHITE = "#FFFFFF";
+export const WHITE = "#FFFFFF";
```
Comment (toloudis) [inline src/components/Layout.tsx:14]: not sure if there is a shared constants file but this seems like a good candidate for it, if it already exists.  Would be weird to depend on Layout.tsx just to get WHITE defined.

File: src/components/Layout.tsx [hunk 2]
```diff
@@ -7,11 +7,11 @@ const { Content } = AntLayout;
 import Navbar from "./Navbar";
 import "../style/index.sass";
 import useSiteMetadata from "./SiteMetadata";
-import { container } from "../style/layout.module.css";
+const { container } = require("../style/layout.module.css");
```
Comment (toloudis) [inline src/components/Layout.tsx:10]: what is it that makes this change necessary?

File: src/img/tube.svg
Comment (toloudis) [inline src/img/tube.svg:1]: 🧪

File: src/pages/disease-cell-line/AICS-97/index.md [hunk 3]
```diff
@@ -6,6 +6,8 @@ date: 2024-02-10T06:30:07.929Z
 snp: NM_000257.4(MYH7):c.767G>A(p.Gly256Glu)
 parental_line: 75
 status: done
-clones: 3 mutant clones; 2 isogenic controls
+clones:
+    mutants: 3
+    isogenic_controls: 2
```
Comment (toloudis) [inline src/pages/disease-cell-line/AICS-97/index.md:11]: just curious, does gatsby pre-transform this .md data?

**Review by toloudis — APPROVED** [review]
just a few questions reading through the code.  Feel free to ask for a better reviewer of CSS.   It is interesting to see a different type of app, using gatsby and yarn.

File: src/components/Layout.tsx (hunk 2 above)
Comment (meganrm) [reply src/components/Layout.tsx:10]: Gatsby, or at least the version I'm using, doesn't have typescript support for modules. So it wants you to make a d.ts file for each module you make. which seemed silly so I did this

**Pull request closed without merging**